{"uuid":"3856654c-5f2c-4e32-9f48-0b30a30749f8","historyId":"b0715b4b82f71748f5a893dbcb4e4627","fullName":"com.WebAutomationProject.SeleniumBasic.ex_15_MiniProjectsuingWaits.verify_error_Message","labels":[{"name":"package","value":"com.WebAutomationProject.SeleniumBasic.ex_15_MiniProjectsuingWaits"},{"name":"testClass","value":"com.WebAutomationProject.SeleniumBasic.ex_15_MiniProjectsuingWaits"},{"name":"testMethod","value":"verify_error_Message"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"WebAutomationProject"},{"name":"subSuite","value":"com.WebAutomationProject.SeleniumBasic.ex_15_MiniProjectsuingWaits"},{"name":"host","value":"MSI"},{"name":"thread","value":"17316@MSI.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"owner","value":"sanket"},{"name":"severity","value":"normal"}],"links":[],"name":"verify_error_Message","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Your email, password, IP address or location did not match] but found [[[EdgeDriver: MicrosoftEdge on windows (7dbfa99e2b51b07123022bbe675caf66)] -> id: js-notification-box-msg]]","trace":"java.lang.AssertionError: expected [Your email, password, IP address or location did not match] but found [[[EdgeDriver: MicrosoftEdge on windows (7dbfa99e2b51b07123022bbe675caf66)] -> id: js-notification-box-msg]]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:644)\r\n\tat com.WebAutomationProject.SeleniumBasic.ex_15_MiniProjectsuingWaits.verify_error_Message(ex_15_MiniProjectsuingWaits.java:50)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Verify that the error message should be display for invalid inputs","steps":[],"attachments":[],"parameters":[],"start":1748171928739,"stop":1748171936774}